Update manage-industry-news.md (#3224)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/manage-industry-news.md

## What your users will see

> Signed-in users that come to the Bing homepage will see your industry news feed under the personalized info from your organization.  

Reviewer: ```suggestion
Signed-in users who go to the Bing homepage will see your industry's news feed under the personalized info for your organization.  
```
